core/domain/services/graphql/connection_resolvers/AbstractConnectionResolver.php

```diff
@@ -34,6 +35,25 @@ protected function getLimit()
         return $limit;
     }
 
+    /**
+     * Determine whether or not the the offset is valid, i.e the entity corresponding to the
+     * offset exists. Offset is equivalent to entity ID. So this function is equivalent to
```

Comment:
didn't knew that `Offset is equivalent to entity ID`

Reply:
In cursor based pagination (use by GraphQL/Relay), offset is always an entity ID which makes things faster.